index.js

```diff
@@ -9,4 +9,9 @@ app.get('/goodbye', function (req, res) {
   res.send('Hello Goodbye')
 })
 
+// app.use(express.static('landing.html'));
```

Comment:
Don't merge commented out code in to master branch.  Especially code that we know doesn't work.